### PR TITLE
Single item not allowed in match

### PR DESCRIPTION
**Pull request by codingblazer**
Now Single item is not allowed in Match template since it is illogical.

### CODE REVIEW

File: source-code/app/src/main/java/org/buildmlearn/toolkit/activity/TemplateEditor.java [hunk 1]
```diff
@@ -826,6 +834,11 @@ private String saveDraft() {
                 Toast.makeText(this, "Unable to perform action: No Meta Details", Toast.LENGTH_SHORT).show();
                 return null;
             }
+            if(templateId==7 && selectedTemplate.getItems(doc).size() == 2)
```
Comment (anuradhagupta345) [inline source-code/app/src/main/java/org/buildmlearn/toolkit/activity/TemplateEditor.java]: @codingblazer Code formatting needed here!

Comment (opticod) [inline source-code/app/src/main/java/org/buildmlearn/toolkit/activity/TemplateEditor.java]: Thanks @anuradhagupta345. 
@codingblazer Please reformat the code.

**Comment by codingblazer** [comment]
@opticod Formatting done. Please review :-)

File: source-code/app/src/main/java/org/buildmlearn/toolkit/activity/TemplateEditor.java [hunk 2]
```diff
@@ -826,6 +834,11 @@ private String saveDraft() {
                 Toast.makeText(this, "Unable to perform action: No Meta Details", Toast.LENGTH_SHORT).show();
                 return null;
             }
+            if(templateId == 7 && selectedTemplate.getItems(doc).size() == 2)
```
Comment (opticod) [inline source-code/app/src/main/java/org/buildmlearn/toolkit/activity/TemplateEditor.java]: We follow 
```
 if () {

}
``` 
formatting.
You can automatically do it in Android studio.

**Comment by codingblazer** [comment]
@opticod Reformatting Done. Please review.

**Pull request merged**